src/activitypub/fediverse-bridge.ts

```diff
@@ -65,12 +65,15 @@ export class FediverseBridge {
                 content: post.content,
                 summary: null,
                 published: Temporal.Now.instant(),
-                attachments: post.imageUrl
-                    ? [
-                          new Image({
-                              url: post.imageUrl,
-                          }),
-                      ]
+                attachments: post.attachments
+                    ? post.attachments
+                          .filter((attachment) => attachment.type === 'Image')
```

Comment:
🤔 What other attachments would there be? and why wouldn't we want to include them on the note?